src/services/passwordRecovery/hooks/index.js

```diff
@@ -18,6 +21,35 @@ const hashId = (hook) => {
 	}
 };
 
+const sendInfo = (hook) => {
+	if (hook.path === 'passwordRecovery') {
+		hook.app.service('/accounts').get(hook.data.account, {
```

Comment:
return hook.app.service('/accounts').get(hook.data.account, { [...]
to pass the hook explicit not implicit by return undefined.

Reply:
So you want to return hook in case the if statement is not true?